platform/config/imports/componentReference.json

```diff
@@ -1,4 +1,31 @@
 {
 	"only": [],
-	"fetchFromMaster": ["amp-youtube", "amp-access-laterpay", "amp-app-banner", "amp-script", "amp-carousel", "amp-consent", "amp-bind", "amp-story", "amp-subscriptions", "amp-list", "amp-script"]
+	"fetchFromMaster": [
```

Comment:
Is somebody already taking care of adding `bento: true` to their frontmatter in ampproject/amphtml?

Reply:
yes, all bento docs will be updated by the team